.gitignore

```diff
@@ -88,6 +88,11 @@ docGen/
 
 # Release outputs
 release/
+
+# e2e tests outputs
 e2e-tests/test-results/
 e2e-tests/playwright-report/
 e2e-tests/playwright/.cache/
+
+# Snapshots generated while running e2e tests locally
+*win*.png
```

Comment:
we dont want locally generated snapshots in the repo, the pipeline will generate linux snapshots and the tests will be validated against them

Reply:
I think here we can be more specific with the path, so as to not exclude **all** filenames that match that pattern : `e2e-tests/tests/snapshots/**/*win*/png`